### PR TITLE
chore: changing exports

### PR DESCRIPTION
**Pull request by ccedrone**
I believe I covered all of the components appropriately, but I need the experts.

### CODE REVIEW

**Review by joasmi723 — APPROVED** [review]

File: packages/react-magma-dom/src/components/Breadcrumb/index.tsx [hunk 1]
```diff
@@ -1,61 +1,2 @@
...
-          <StyledList isInverse={isInverse} ref={ref} theme={theme}>
-            {children}
-          </StyledList>
-        </nav>
-      </BreadCrumbContext.Provider>
-    );
-  }
-);
+export * from './Breadcrumb';
+export * from './Item';
```
Comment (aaronmaturen) [inline packages/react-magma-dom/src/components/Breadcrumb/index.tsx:2]: Not sure if this matters in any schema of things... should we do `Item` or `BreadcrumbItem` ?

Comment (joasmi723) [reply packages/react-magma-dom/src/components/Breadcrumb/index.tsx:2]: `BreadcrumbItem` would be preferable, but that would be breaking change. We should change it to BreadcrumbItem for v3.

Comment (joasmi723) [reply packages/react-magma-dom/src/components/Breadcrumb/index.tsx:2]: Actually, I _do_ think it should be `BreadcrumbItem`, so I think you're correct.  We'll look at that.

Comment (joasmi723) [reply packages/react-magma-dom/src/components/Breadcrumb/index.tsx:2]: Ah, the component is named BreadcrumbItem, which is how it should be and is how it gets exported.  But the file is named Item.tsx, so I guess that should change.

**Pull request merged**